SConstruct

```diff
@@ -1067,24 +1071,45 @@ if env['python_package'] in ('full','default'):
 
     try:
         info = getCommandOutput(env['python_cmd'], '-c', script)
-        (env['python_version'], env['python_usersitepackages']) = info.splitlines()[-2:]
     except OSError as err:
         if env['VERBOSE']:
             print 'Error checking for Python 2:'
             print err
-        info = False
+        warn_no_python = True
+    except subprocess.CalledProcessError as err:
+        if env['VERBOSE']:
+            print 'Error checking for Python 2:'
+            print err, err.output
+        warn_no_python = True
+    else:
+        (env['python_version'], numpy_version,
+         env['python_usersitepackages']) = info.splitlines()[-3:]
+        numpy_version = LooseVersion(numpy_version)
+        if numpy_version < numpy_min_version:
+            python_message += ("NumPy for Python {0} not found or incompatible version: "
+                               "Found {1} but {2} or newer is required.\n".format(
+                                   env['python_version'], numpy_version, numpy_min_version))
+            warn_no_python = True
+        else:
+            print 'INFO: Using NumPy version {0} for Python {1}.'.format(
+                numpy_version, env['python_version'])
 
-    if not info:
+    if warn_no_python:
         if env['python_package'] == 'default':
             print ('WARNING: Not building the full Python 2 package because the Python '
                    '2 interpreter %r could not be found or a required dependency '
                    '(e.g. numpy) was not found.' % env['python_cmd'])
+            if env['VERBOSE']:
```

Comment:
The main reason I think it's useful to always print this kind of info is because of the number of support requests that we get where the first step is invariably 'please rerun with `verbose=y`'.